client/src/components/DataFiles/DataFilesModals/DataFilesCopyModal.js

```diff
@@ -80,6 +81,28 @@ const DataFilesCopyModal = React.memo(() => {
       }
     });
   };
+  const modifiableUserData =
```

Comment:
Can we remove this logic from the copy modal and instead pass the permission information in as props from the toolbar? The reason behind my thinking is that some code (such as computing `modifiableUserData`) now lives in 2 places and we could introduce bugs if we change the logic in one place but forget to change it in the other.